Sources/MonkeyKing/Helpers.swift

```diff
@@ -26,13 +26,17 @@ extension MonkeyKing {
         ]
 
         guard let accessTokenAPI = urlComponents?.string else {
-            completionHandler(["code": code], nil, nil)
+            completionHandler(.failure(.sdk(.invalidURLScheme)))
             return
         }
 
         // OAuth
-        shared.request(accessTokenAPI, method: .get) { json, response, error in
-            completionHandler(json, response, error)
+        shared.request(accessTokenAPI, method: .get) { json, _, error in
+            if error != nil {
+                completionHandler(.failure(.apiRequest(.unrecognizedError(response: nil))))
```

Comment:
Maybe we should pass the error.
e.g. `completionHandler(.failure(.apiRequest(. underlying(error: error, response: json))))`

Reply:
OK.